app/components/avo/tab_group_component.html.erb

```diff
@@ -10,7 +10,8 @@
   } do %>
   <% visible_tabs.each_with_index do |tab, index| %>
     <%= content_tag :div, **args(tab) do %>
-      <div class="border rounded-lg p-2 -mx-2 -my-2 lg:p-4 lg:-mx-4 lg:-my-4 space-y-4">
+      <%= render Avo::PanelComponent.new name: group.title, description: group.description %>
```

Comment:
Can we extract the header section from the `Avo::PanelComponent` and reuse it here? I.E Having a `Avo::PanelHeaderCompontent` and render it here and inside `Avo::PanelComponent`?